applications/openshift/general/var_backup_solution_crds_regex.var

```diff
@@ -0,0 +1,15 @@
+documentation_complete: true
+
+title: 'Known CRDs which are provided by backup solutions'
+
+description: |-
+   'A regular expression that lists all CRDs that are known to be part of a backup solution'
+
+type: string
+
+operator: equals
+
+interactive: false
```

Comment:
I think this doesn't matter much for the Compliance Operator., but technically this should be `interactive: true`.
So that any tooling that may follow the SCAP standard knows that this variable can be edited by the user/admin.